### ISSUE TITLE
Error compiling on elixir 1.10

### ISSUE THREAD

**Issue by ConnorRigby** [membraneframework/shmex#10]
```
== Compilation error in file lib/shmex_native.ex ==
** (ArgumentError) could not call Module.delete_attribute/2 because the module Shmex.Native is in read-only mode (@after_compile)
    (elixir 1.10.2) lib/module.ex:2093: Module.assert_not_readonly!/2
    (elixir 1.10.2) lib/module.ex:1285: Module.delete_attribute/2
    (bundlex 0.2.7) lib/bundlex/loader.ex:45: Bundlex.Loader.__after_compile__/2
    (stdlib 3.11.2) lists.erl:1263: :lists.foldl/3
    (stdlib 3.11.2) erl_eval.erl:680: :erl_eval.do_apply/6

```

**Comment by bblaszkow06** [comment]
Thanks for the report! It seems it's a bug in Bundlex and before 1.10 `Module.delete_attribute/2` failed silently -> https://github.com/membraneframework/bundlex/issues/54

**Comment by bblaszkow06** [comment]
Fixed in 34507e49029639fa60b7105ac0a2d4464c00ab8f and released as v0.2.1
